### ISSUE TITLE
Extension Error: Parameter 'url' must be a string, not undefined

### ISSUE THREAD

**Issue by Onset** [GoogleChrome/lighthouse#1071]
**Lighthouse Version**: 1.2.1
**Chrome Version**: 54.0.2840.99
**Error Message**: Parameter 'url' must be a string, not undefined
**Stack Trace**:
 ```
TypeError: Parameter 'url' must be a string, not undefined
    at Url.parse (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:41949:11)
    at Object.urlParse [as parse] (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:41943:5)
    at artifacts.ConsoleTimeUsage.usage.filter.err (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:808:18)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:807:54)
    at chain.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11082:24)
```

**Comment by ebidel** [comment]
Can you share the URL you tested against?

**Comment by Onset** [comment]
http://onset.github.io/lighthouse-test/

**Comment by ebidel** [comment]
The root cause is that the usage of `console.time()` here is in eval'd code and the DT protocol doesn't give us the url for this case. 

https://bugs.chromium.org/p/chromium/issues/detail?id=646849

**Comment by Onset** [comment]
Thank you for your time and the explanation.

**Comment by brendankenny** [comment]
fixed in #1073
